### PR TITLE
Disable form if user is not admin

### PR DESCRIPTION
**Pull request by AdityaKhatri**
* Add migration in redux to remove siloDomaindata.connectorsView

Merge: https://github.com/toggle-corp/react-store/pull/649

### CODE REVIEW

File: src/views/Connector/Details/Form/connector-utils.js [hunk 1]
```diff
@@ -68,6 +68,7 @@ export const getUsersTableHeader = (handleToggleUserRoleClick, handleDeleteUserC
         order: 5,
         modifier: (row) => {
             const isAdmin = row.role === 'admin';
+            const isSelf = row.user === userId;
```
Comment (tnagorra) [inline src/views/Connector/Details/Form/connector-utils.js:76]: I don't think we should do this, as this action can be done through select input

File: src/components/other/FilterLeadsForm/index.js [hunk 2]
```diff
@@ -56,12 +56,14 @@ const propTypes = {
 
     // eslint-disable-next-line react/forbid-prop-types
     leadOptionsRequest: PropTypes.object.isRequired,
+    onlyUnprotected: PropTypes.bool,
```
Comment (frozenhelium) [inline src/components/other/FilterLeadsForm/index.js]: A better prop name would be more comprehensive

File: src/views/Export/index.js [hunk 3]
```diff
@@ -71,11 +79,68 @@ const propTypes = {
...
+        method: requestMethods.POST,
+        onMount: true,
+        query: ({
+            fields: ['id', 'title', 'created_at'],
+        }),
+        body: ({
+            props: {
+                projectId,
+                filters,
+                projectRole: {
```
Comment (frozenhelium) [inline src/views/Export/index.js:99]: Is project role always defined?

File: src/views/Export/index.js [hunk 4]
```diff
@@ -71,11 +79,68 @@ const propTypes = {
...
+                    exportPermissions = {},
+                },
+            },
+        }) => {
+            const onlyUnprotected = exportPermissions.create_only_unprotected;
+            const sanitizedFilters = getFiltersForRequest(filters);
+
+            if (onlyUnprotected) {
```
Comment (frozenhelium) [inline src/views/Export/index.js]: We should add a comment here, no?

File: src/views/Export/index.js (hunk 3 above)
Comment (AdityaKhatri) [reply src/views/Export/index.js:99]: yes

File: src/views/Export/index.js (hunk 4 above)
Comment (AdityaKhatri) [reply src/views/Export/index.js]: No

File: src/views/Export/index.js [hunk 5]
```diff
@@ -455,8 +459,15 @@ export default class Export extends React.PureComponent {
...
             geoOptions,
+            projectRole: {
+                exportPermissions = {},
+            },
+            leadsGetRequest: {
+                pending: pendingLeads,
+            },
         } = this.props;
         const { filters } = analysisFramework || {};
+        const onlyUnprotected = exportPermissions.create_only_unprotected;
```
Comment (frozenhelium) [inline src/views/Export/index.js]: snake case looks ugly

Comment (AdityaKhatri) [reply src/views/Export/index.js]: Clean your glasses

**Pull request merged**